README.md

```diff
@@ -288,6 +288,7 @@ A curated list of awesome Java frameworks, libraries and software.
 
 * [Apache DeltaSpike](https://deltaspike.apache.org/) - CDI extension framework.
 * [Dagger2](http://google.github.io/dagger/) - Compile-time injection framework without reflection.
+* [Feather](https://github.com/zsoltherpai/feather) - Lightweight dependency injection for Java and Android (JSR-330).
```

Comment:
See contribution rules:

1. If an entry has a similar scope as other entries in the same category, the description must state the unique features that distinguishes it from the other entries.